addons/event/models/event_event.py

```diff
@@ -99,6 +99,10 @@ class EventEvent(models.Model):
     _inherit = ['mail.thread', 'mail.activity.mixin']
     _order = 'date_begin'
 
+    @property
+    def _autovacuum(self):
+        return super()._autovacuum + ('_clean_pipeline',)
```

Comment:
In retrospect, I find this boilerplate much less elegant than using a naming convention.
I suggest to use names like `_autovacuum_XXX` for autovacuum methods.
The cost of retrieving those methods by introspecting models is pretty small.

In this case, `_autovacuum_ended_events` looks clear an intuitive.